ConsoleApp/Board.cs

```diff
@@ -0,0 +1,177 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+
+namespace SnakeGame;
+public class Board
+{
+    public static DirectionMap DirectionMap { get; } = new DirectionMap();
+
+    private readonly string instructions = "How to Play: Avoid hitting walls or yourself. Grow by eating food (Y). Highest length wins.";
```

Comment:
code convention violated
```suggestion
    private readonly string _instructions = "How to Play: Avoid hitting walls or yourself. Grow by eating food (Y). Highest length wins.";
```